### PR TITLE
feat: Added SpO2 in community nurses log update form

### LINKED ISSUE #8865: Add Spo2 on vitals section of Community Nurse log update
**Is your feature request related to a problem? Please describe.**
Add Spo2 vital field on the community nurses log update form. Nurses from the field were forced to file another brief update because Spo2 was available there. 

Add the Spo2 field here,
Image

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by noufalrahim**
## Proposed Changes

- Fixes #8865 
- Added Sp02 field in community nurse log update form

@ohcnetwork/care-fe-code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.ohc.network).
- [ ] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

### CODE REVIEW

File: src/Components/Patient/DailyRounds.tsx [hunk 1]
```diff
@@ -788,6 +788,30 @@ export const DailyRounds = (props: any) => {
...
+                thresholds={[
+                  {
+                    value: 0,
+                    className: "text-danger-500",
+                    label: "Low",
+                  },
+                  {
+                    value: 90,
+                    className: "text-primary-500",
+                    label: "Normal",
```
Comment (rithviknishad) [inline src/Components/Patient/DailyRounds.tsx]: i18n

File: src/Components/Patient/DailyRounds.tsx [hunk 2]
```diff
@@ -788,6 +788,30 @@ export const DailyRounds = (props: any) => {
...
+                {...field("ventilator_spo2")}
+                unit="%"
+                start={0}
+                end={100}
+                step={1}
+                thresholds={[
+                  {
+                    value: 0,
+                    className: "text-danger-500",
+                    label: "Low",
```
Comment (rithviknishad) [inline src/Components/Patient/DailyRounds.tsx]: i18n

File: src/Components/Patient/DailyRounds.tsx [hunk 3]
```diff
@@ -788,6 +788,30 @@ export const DailyRounds = (props: any) => {
...
+                  },
+                  {
+                    value: 90,
+                    className: "text-primary-500",
+                    label: "Normal",
+                  },
+                  {
+                    value: 100,
+                    className: "text-danger-500",
+                    label: "High",
```
Comment (rithviknishad) [inline src/Components/Patient/DailyRounds.tsx]: i18n

File: .hintrc
Comment (rithviknishad) [inline .hintrc:1]: why was this added?

**Review by rithviknishad — CHANGES_REQUESTED** [review]

**Comment by rithviknishad** [comment]
Please follow PR's merge checklist

File: src/Components/Patient/DailyRounds.tsx (hunk 1 above)
Comment (noufalrahim) [reply src/Components/Patient/DailyRounds.tsx]: `    thresholds={[
                  {
                    value: 0,
                    className: "text-danger-500",
                    label: t("low"),
                  },
                  {
                    value: 90,
                    className: "text-primary-500",
                    label: t("normal"),
                  },
                  {
                    value: 100,
                    className: "text-danger-500",
                    label: t("high"),
                  },
                ]}
              />`
              
              Is this fine?

Comment (rithviknishad) [reply src/Components/Patient/DailyRounds.tsx]: yup

Comment (noufalrahim) [reply src/Components/Patient/DailyRounds.tsx]: Screenshot from 2024-10-23 10-32-41

File: .hintrc
Comment (noufalrahim) [reply .hintrc:1]: Not sure how it got added.. I will remove it..

**Comment by noufalrahim** [comment]
Modified
Screenshot from 2024-10-23 14-59-57

**Review by rithviknishad — APPROVED** [review]

**Comment by noufalrahim** [comment]
Opened another PR
